migrate/loader.go

```diff
@@ -0,0 +1,137 @@
+package migrate
+
+import (
+	"fmt"
+	"github.com/packwiz/packwiz/core"
+	"github.com/spf13/cobra"
+	"golang.org/x/exp/slices"
+	"os"
+	"strings"
+)
+
+var loaderCommand = &cobra.Command{
+	Use:   "loader",
+	Short: "Migrate your loader versions to newer versions.",
+	Args:  cobra.ExactArgs(1),
+	Run: func(cmd *cobra.Command, args []string) {
+		modpack, err := core.LoadPack()
+		if err != nil {
+			// Check if it's a no such file or directory error
+			if os.IsNotExist(err) {
+				fmt.Println("No pack.toml file found, run 'packwiz init' to create one!")
+				os.Exit(1)
+			}
+			fmt.Printf("Error loading pack: %s\n", err)
+			os.Exit(1)
+		}
+		// Get our current loader
+		currentLoader := modpack.GetLoaders()
+		// Do some sanity checks on the current loader slice
+		if len(currentLoader) == 0 {
+			fmt.Println("No loader is currently set in your pack.toml!")
+			os.Exit(1)
+		} else if !slices.Contains(currentLoader, "quilt") && len(currentLoader) > 1 {
```

Comment:
Edge case: if Quilt is being used, the check for more than 1 loader is ignored (so the user could have 3 loaders?)